src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/model/out/SuccessfulResponse.java

```diff
@@ -2,12 +2,12 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class SuccessfulReconciliationResponse {
+public class SuccessfulResponse {
 
     @JsonProperty("id")
```

Comment:
@ApiProperty documentation would be nice

Reply:
Added